deploy/helm/hdfs-operator/crds/crds.yaml

```diff
@@ -57,20 +57,6 @@ spec:
                       format: uint8
                       minimum: 0.0
                       type: integer
-                    listenerClass:
-                      default: cluster-internal
-                      description: |-
-                        This field controls which type of Service the Operator creates for this HdfsCluster:
-
-                        * cluster-internal: Use a ClusterIP service
-
-                        * external-unstable: Use a NodePort service
-
-                        This is a temporary solution with the goal to keep yaml manifests forward compatible. In the future, this setting will control which [ListenerClass](https://docs.stackable.tech/home/nightly/listener-operator/listenerclass.html) will be used to expose the service, and ListenerClass names will stay the same, allowing for a non-breaking change.
-                      enum:
-                        - cluster-internal
-                        - external-unstable
-                      type: string
```

Comment:
Are the CRD changes approved?

Reply:
They were discussed in https://stackable-workspace.slack.com/archives/C02FZ581UCD/p1704714818413279, and this _re_aligns with the original ADR.

I'm fine adding it to the arch discussion list for next week.

Reply:
approved